src/service/iot_config.proto

```diff
@@ -346,14 +353,25 @@ message gateway_load_region_req_v1 {
   bytes signature = 4;
 }
 
-message gateway_load_region_res_v1 {}
+message admin_load_region_res_v1 {}
 
-message gateway_location_req_v1 {
-  bytes gateway = 1;
+message admin_add_key_req_v1 {
+  enum key_type_v1 {
+    operator = 0;
+    packet_router = 1;
+  }
+
+  string pubkey = 1;
+  key_type_v1 key_type = 2;
+  bytes signature = 3;
```

Comment:
Document this field stating that this request needs to be signed by an existing admin key (vs the pub key being added)

Reply:
added [ca3da06](https://github.com/helium/proto/pull/288/commits/ca3da06d51faa7b490e2377cf47c20098e9900c5)